src/sentry/static/sentry/app/views/userFeedback/organizationUserFeedback.jsx

```diff
@@ -63,10 +66,41 @@ class OrganizationUserFeedback extends AsyncView {
     return this.renderResults();
   }
 
+  renderIntegrationHint() {
+    const {organization, selection} = this.props;
+
+    const allAccessibleProjects = organization.projects.filter(
+      project => project.isMember
+    );
+
+    const hasSingleProject =
+      selection.projects.length === 1 ||
+      (selection.projects === 0 && allAccessibleProjects.length === 1);
+
+    if (!hasSingleProject) {
```

Comment:
Hmm I think in this case we can use `app/actionCreators/navigateTo` to ask the user what project when navigating to setup...

You can add a link with a clickhandler that calls `navigateTo('/settings/:orgId/projects/:projectId/user-feedback/`, router)` and it'll open a modal to ask you to pick a project.

Reply:
We may need to update ContextPickerModal to filter projects by `isMember`